src/CMakeLists.txt

```diff
@@ -122,6 +122,10 @@ else()
   set(ABC_LIBRARY "libabc")
 endif()
 
+if(DEBUG_DRT_UNDERFLOW)
+  add_definitions(-DDEBUG_DRT_UNDERFLOW=1)
+endif()
```

Comment:
Can this be moved in drt and use target_compile_definitions since this isn't affecting anything else?